wiki/Tournaments/MWC/2018/7K/en.md

```diff
@@ -0,0 +1,211 @@
+
+## Ruleset
+### Tournament Rules
+
+1. The osu!mania 7K World Cup is a country-based team tournament, played on the osu!mania game mode.
+   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the amount of incoming registrations.**
+
+2. Beatmap scoring is based on Score V2.
+
+3. The beatmaps for each round will be announced by the mapset selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
+   - The Group Stage mappool will be announced after the drawings.
+   - One beatmap will be given as a tiebreaker beatmap. This beatmap will only be played in case of a tie.
+
+4. Match schedule will be settled by the Tournament Management (see below).
+5. If no staff or referee is available, the match will be postponed.
+6. Failed players scores do not get added to the team score.
+   - Reviving and surviving during a beatmap gets considered as passing it.
+
+7. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
+8. If the beatmap ends in a draw, the game will be nullified.
+9. If a player disconnects, they get treated as if they failed the beatmap.
+   - Disconnects within 30 seconds after beatmap begin can be rematched. This is up to the referee's discretion. The played beatmap might be aborted for this.
+
+10. Beatmaps cannot be reused in the same match unless the game was nullified.
+11. **The size of each team is restricted to a maximum of 4 players and a minimum of 2.**
+   - This might change depending on the amount of registrations.
+11. If less than the minimum required players attend, the maximum time the match can be postponed is 10 minutes.
+12. Exchanging players during a match is allowed without limitations.
+13. Lag is not a valid reason to nullify a beatmap.
+14. All players are supposed to keep the match running fluent and without delays. Excessive match delays coming from the player's side can be issued with penalties.
+15. If a player disconnects between the beatmaps and the team can not provide an exchange, the match can be delayed 10 minutes at maximum.
+16. All players and referees must to be treated with respect. Instructions of the referees and tournament management is to be followed. Decisions labeled as final are not to be objected.
+17. Disrupting the match by foul play, picking inappropriate warmup beatmaps (see below), insulting and provoking other players or referees, delaying the match or other deliberate inappropriate misbehavior is strictly prohibited.
+18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.
+
+19. In Group stage, 'Win by default' will be considered as win by 4:0, +1.0 score difference ratio.
```

Comment:
replace _In Group stage,_ with "In the Group Stage,"
replace  _'Win by default'_ with "a 'Win by Default'"
add an "a" between _as_ and _win_, i.e "as a win"